torch/fx/node.py

```diff
@@ -220,6 +238,54 @@ def __update_args_kwargs(self, new_args : Tuple['Argument', ...], new_kwargs : D
     def __repr__(self) -> str:
         return self.name
 
+    def _pretty_print_target(self, target):
+        """
+        Make target printouts more user-friendly.
+        1) builtins will be printed as `builtins.xyz`
+        2) operators will be printed as `operator.xyz`
+        3) other callables will be printed with qualfied name, e.g. torch.add
+        """
+        if isinstance(target, str):
+            return target
+        if hasattr(target, '__module__'):
+            if not hasattr(target, '__name__'):
+                # Just to be defensive, if we don't have `__name__`, get the
+                # qualname. Not sure if this happens for any members of `operator`
+                # or `builtins`. This fallback path is not as good, since e.g.
+                # things in `operator` have `_operator` as their __module__.
+                return _get_qualified_name(target)
+            if target.__module__ == 'builtins':
+                return f'builtins.{target.__name__}'
+            elif target.__module__ == '_operator':
+                return f'operator.{target.__name__}'
+        return _get_qualified_name(target)
+
+    def format_node(self, placeholder_names=None, maybe_return_typename=None) -> Optional[str]:
```

Comment:
Please add type annotations and `Args:` to the doc string. Also make it clear that both of these are output parameters